bundles/org.openhab.binding.asuswrt/src/main/java/org/openhab/binding/asuswrt/internal/structures/AsuswrtConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.asuswrt.internal.structures;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link AsuswrtConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Christian Wild - Initial contribution
+ */
+@NonNullByDefault
+public class AsuswrtConfiguration {
+
+    /* THING CONFIGUTATION PROPERTYS */
```

Comment:
It's would be nicer without typos and proper capitalization. Please also fix the others.

```suggestion
    /* Thing configuration properties */
```